client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -677,6 +678,17 @@ export class WorkspaceView extends React.Component<any, any> {
       ? this.getAvailablePointers(workspace, rootWorkspaceScratchPad)
       : this.getAvailablePointers(workspace);
 
+    const shouldShowJudgeTextFromScratchpad =
+      hasParent &&
+      !isOracleWorkspace &&
+      doesRootWorkspaceScratchpadContainRelevantContent;
+
+    const judgeRegex = /(?<=<judge>)(.*)(?=<judge>)/;
+    const judgeTextArray = judgeRegex.exec(
+      listOfSlateNodesToText(_.get(rootWorkspaceScratchPad, "value"), false),
+    );
+    const judgeText = judgeTextArray ? judgeTextArray[0] : null;
```

Comment:
So if you change the regex as mentioned above, the `judgeTextArray[0]` should be `judgeTextArray[1]`.